web/src/admin/policies/geoip/GeoIPPolicyForm.ts

```diff
@@ -0,0 +1,170 @@
+                                return item.name;
+                            }}
+                            .selected=${(item: TypeCreate): boolean => {
+                                return this.instance?.asnMode === item.name;
+                            }}
+                        >
+                        </ak-search-select>
+                        <p class="pf-c-form__helper-text">
+                            ${msg(
+                                "Block or only allow connections from IPs from the provided autonomous systems.",
+                            )}
+                        </p>
+                    </ak-form-element-horizontal>
+                    <ak-form-element-horizontal label=${msg("ASN List")} name="asnList">
+                        <input
+                            type="text"
+                            value="${ifDefined(this.instance?.asnList || "")}"
+                            class="pf-c-form-control"
+                        />
+                        <p class="pf-c-form__helper-text">
+                            ${msg(
+                                "List of autonomous system numbers. Comma separated. E.g. 13335,15169,20940",
+                            )}
+                        </p>
+                    </ak-form-element-horizontal>
+                    <ak-form-element-horizontal label=${msg("Country Mode")} name="countryMode">
+                        <ak-search-select
+                            .fetchObjects=${async () => {
+                                return [
+                                    { name: "allow", label: "Allow" },
+                                    { name: "block", label: "Block" },
+                                ];
+                            }}
+                            .renderElement=${(item: App): string => {
+                                return item.label;
+                            }}
+                            .value=${(item: App | undefined): string | undefined => {
+                                return item.name;
+                            }}
+                            .selected=${(item: App): boolean => {
+                                return this.instance?.countryMode === item.name;
+                            }}
+                        >
+                        </ak-search-select>
+                        <p class="pf-c-form__helper-text">
+                            ${msg(
+                                "Block or only allow connections from IPs from the provided countries.",
+                            )}
+                        </p>
+                    </ak-form-element-horizontal>
+                    <ak-form-element-horizontal label=${msg("Country List")} name="countryList">
+                        <input
+                            type="text"
+                            value="${ifDefined(this.instance?.countryList || "")}"
```

Comment:
Since you've provided a not-`undefined` value for falsity, `ifDefined(this.instance?.countryList || "")` will always be defined.  You can replace that whole expression with `this.instance?.countryList ?? ""`